cmd/openshift/operator/kodata/info/operator-config.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: operators-info
```

Comment:
@PuneetPunamiya I see `operator-config.yaml` is same for both Openshift and kubernetes 
So can you keep it in common place as both are same

Reply:
as this is in `cmd/<platform>/.../kodata` (not `config/<platform>/) let us keep this as separate files. 

otherwise, we will have to setup kustomize hierarchy for `kodata` also like we do for `config/`. I think that will be too much effort for just 1 file. ie, setuping kustomize and related changes in release pipeline to set this file in kodata.

🧑‍💻  In the future, if we have more common files like this, we can think about a script/kustomize to handle this.

Reply:
Okay
Ya agree i mean require some effort to do it